src/value_generators/test/DocumentGeneratorTestCases.yml

```diff
@@ -498,4 +498,40 @@ Tests:
     #    - { "int" : { "$numberLong" : "2000000000"}, "string" : "2000000000" }
     - "string": "2"
 
+  # Now moves with time so would always fail.
+  # - Name: Now
+  #   GivenTemplate:
+  #     date: {^Now: {}}
+  #   ThenReturns:
+  #   - { "date" : { "$date" : { "$numberLong" : "1453689676330" } } }
```

Comment:
Would it be worth extending the test runner to have an approximate check here? Something to make sure it returns a nonZero number?

Reply:
yes but ... not in this ticket.

I have uncommented this test as there is already another failing test in here.